apps/xprof_gui/priv/app/callee_explorer.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+export default class CalleeExplorer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      callees: []
+    };
+  }
+
+  componentDidMount() {
+    this.getCallees();
+  }
+
+  getCallees() {
+    const mfa = this.props.mfa;
+
+    $.ajax({
+      url: "/api/get_callees",
+      data: {
+        mod: mfa[0],
+        fun: mfa[1],
+        arity: mfa[2]
+      }
+    }).done((response) => {
+      this.setState({
+        callees: response
+      });
+    });
+  }
+
+  handleCalleeClick(e) {
+    const calleeQuery = e.target.value;
+
+    $.ajax({
+      url: "/api/mon_start",
+      data: { query: calleeQuery }
+    });
+  }
+
+  render() {
+    const callees = this.state.callees;
+    var output;
+
+    if (callees.length !== 0) {
+      output = callees.map((callee) => {
+        const mfa = `${callee[0]}:${callee[1]}/${callee[2]}`;

```

Comment:
this is not good as this is Erlang syntax and GUI should be syntax-mode agnostic (supporting also Elixir)
Usually what happens is that the backend should format language-sepcific terms (specifically `xprof_core` because that is what will run in the target node) and pass to the frontend as strings

eg "/api/funs" (and `xprof_core_vm_info:get_available_funs/1`) returns a list of query strings

in this case "/api/get_callees" could return a list of label and query string pairs
(this way we can have short labels like "lists:reverse/2" for the buttons but a calleeQuery string that also filters on the caller)
there could be `xprof_core:get_called_funs_pp` which calls current `xprof_core:get_called_funs` and converts it to this list of formatted pairs with `ModeCB:fmt_mfa(Mod, Fun, Arity)`